### DISCUSSION TITLE
iOS App is Native or Cross Platform

### DISCUSSION THREAD

**Discussion by kamikazeeGodzilla** [nammayatri/nammayatri#8029] [General]
I wanted to know the app is in native or its hybrid app for iOS. As I can use the app it seems that its hybrid App.

**Comment by witcher-shailesh** [comment]
Hi @kamikazeeGodzilla - it's a hybrid app

**Reply by kamikazeeGodzilla** [reply]
I saw there is a lag and bad UI/UX experience, can we create native iOS app for NY. I will contribute for that from scratch. I am an iOS dev in Gojek India.

**Reply by kamikazeeGodzilla** [reply]
Can we have a discussion regarding that please
